zustand/useHighscoresStore.js

```diff
@@ -4,15 +4,38 @@ import { nanoid } from "nanoid";
 import { useState, useEffect } from "react";
 
 const useHighscoresStore = createLocalStorageStore(
-  (set) => ({
+  (set, get) => ({
     highscores: [],
-    addHighscore: (name, time, failed) =>
+    addHighscore: (name, time, formattedTime, failed) => {
       set((state) => ({
         highscores: [
           ...state.highscores,
-          { name: name, time: time, failed: failed, id: nanoid() },
+          {
+            name: name,
+            time: time,
+            formattedTime: formattedTime,
+            failed: failed,
+            id: nanoid(),
+          },
```

Comment:
you could use object shorthand syntax here if you want:

```suggestion
          {
            name,
            time,
            formattedTime,
            failed,
            id: nanoid(),
          },
```